src/types.ts

```diff
@@ -66,5 +67,9 @@ export interface Context extends Rule.RuleContext {
     browsers?: Array<string>;
     polyfills?: Array<string>;
     lintAllEsApis?: boolean;
+    browserslistOpts?: BrowsersListOpts;
   };
 }
+
+export interface BrowsersListOpts
+  extends Exclude<DefaultBrowsersListOpts, "path"> {}
```

Comment:
I decided to exclude `path` since that is already configured in the `opts` [above](https://github.com/amilajack/eslint-plugin-compat/pull/587/files#diff-ca5696b367d474e785317cb7a0d9853fef5729387ab8d0fab4c46268c03aae99R187), but I can remove that if you want.

I also considered wrapping it in `Partial<>`; it appears the browserslist type already has everything as optional, so I decided against it for now, but if you want I can add it in if you want to be extra sure that nothing is ever "required" in that config.

Reply:
Also, maybe you only want devs to pass `env` and no other options? If so, let me know and I'll just do that, too.

Reply:
That makes sense, thanks for the context here. I like the reasoning here, let's go with this implementation